Tests/DatabaseServiceTests/Repositories/UserRepositoryTests.cs

```diff
@@ -0,0 +1,125 @@
+﻿using System;
+
+using Microsoft.EntityFrameworkCore;
+
+using NUnit.Framework;
+using Moq;
+
+using DTO;
+using DataBaseService.Database;
+using DataBaseService.Database.Models;
+using DataBaseService.Repositories;
+using DataBaseService.Repositories.Interfaces;
+using DataBaseService.Mappers.Interfaces;
+
+
+namespace DatabaseServiceTests.Repositories
+{
+    public class UserRepositoryTests
+    {
+        IUserRepository repository;
+        Mock<IUserMapper> mapper;
```

Comment:
Why we need this mock in those tests?

Reply:
because mapper must be is mock

Reply:
We can add Users in our db without using mapper